fix_flow_studio_ui_ids.py

```
@@ -0,0 +1,18 @@
+with open("tests/test_flow_studio_ui_ids.py", "r") as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)